CBLClient/PeerToPeer.py

```diff
@@ -93,6 +95,37 @@ def server_stop(self, url_listener, end_point_type):
         args.setString("endPointType", end_point_type)
         return self._client.invokeMethod("peerToPeer_serverStop", args)
 
+    def configureCollection(self, host, server_db_name, port=5000, continuous=None, authenticator=None, replication_type=None, endPointType="MessageEndPoint", basic_auth=None,
```

Comment:
This is quite a big list of arguments. Cld this list be reduced? A big list of arguments can cause conflicting argument behaviour, amongst others. 

Possible options to reduce could include splitting one general function to smaller functions and remove some of the arguments that are not currently being used.

Reply:
I have removed the parameters that are not needed at this point of time.